main_test.go

```diff
@@ -148,6 +148,11 @@ func TestEndpoints(t *testing.T) {
 		{"/search?prerelease=true&discovery=datasets:good_content.errors&discovery=fields:process.path", "/search", "search-discovery-multiple-no-match.txt", searchHandler},
 		{"/favicon.ico", "", "favicon.ico", faviconHandler},
 
+		{"/search?package=agent_version&agent.version=9.1.0", "/search", "search-agent-910.json", searchHandler},
+		{"/search?package=agent_version&agent.version=9.5.0", "/search", "search-agent-950.json", searchHandler},
+		{"/categories?package=agent_version&agent.version=9.1.0", "/categories", "categories-agent-910.json", categoriesHandler},
+		{"/categories?package=agent_version&agent.version=9.5.0", "/categories", "categories-agent-950.json", categoriesHandler},
+
```

Comment:
Categories endpoint does not have a `package` query parameter ([link](https://github.com/elastic/package-registry/blob/d61a59f81c64063c3fe626bd5e28e1e1f0d4393b/categories.go#L158)):

```suggestion
		{"/categories?agent.version=9.1.0", "/categories", "categories-agent-910.json", categoriesHandler},
		{"/categories?agent.version=9.5.0", "/categories", "categories-agent-950.json", categoriesHandler},

```